kv/tikv/storage/interfaces/server.go

```diff
@@ -0,0 +1,20 @@
+package interfaces
+
+import (
+	"github.com/pingcap-incubator/tinykv/kv/pd"
+	"github.com/pingcap-incubator/tinykv/kv/tikv/dbreader"
+	"github.com/pingcap-incubator/tinykv/kv/tikv/inner_server"
+	"github.com/pingcap-incubator/tinykv/proto/pkg/kvrpcpb"
+	"github.com/pingcap-incubator/tinykv/proto/pkg/tikvpb"
+)
+
+// InnerServer represents the internal-facing server part of TinyKV, it handles sending and receiving from other
+// TinyKV nodes. As part of that responsibility, it also reads and writes data to disk (or semi-permanent memory).
+type InnerServer interface {
```

Comment:
It's too weird to define `InnerServer` in storage.  But seems necessary now, it's can be moved back to server.go once scheduler is removed.